### PR TITLE
Exports new permision settings for regional admins

### LINKED ISSUE #4901: br and mx admin roles shouldn't have access to configuration or users admin menu items
remove permissions from regional admin roles 
- [x] view user profiles
- [x] translate interface texts

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

Removes permissions from regional admins
- access user profiles
- translate interface 
#### How should this be manually tested?

Create a new MR or BR admin and test you don't have these permissions 
#### What are the relevant tickets?

Fixes most of #4901 -- waiting till andrea gets back to finish it out
#### Questions

Andrea left this group of things in a comment, but I can't find matching permissions for any of them. I have no idea what the intent of the comment was, 
https://github.com/DoSomething/phoenix/issues/4901#issuecomment-138592399

### CODE REVIEW

File: lib/modules/dosomething/dosomething_user/dosomething_user.views_default.inc [hunk 1]
```diff
@@ -305,11 +305,11 @@ function dosomething_user_views_default_views() {
...
-  $handler->display->display_options['fields']['source']['id'] = 'source';
-  $handler->display->display_options['fields']['source']['table'] = 'entity_translation';
-  $handler->display->display_options['fields']['source']['field'] = 'source';
-  $handler->display->display_options['fields']['source']['relationship'] = 'entity_translations';
+  /* Field: Content: Language */
+  $handler->display->display_options['fields']['language_1']['id'] = 'language_1';
+  $handler->display->display_options['fields']['language_1']['table'] = 'node';
+  $handler->display->display_options['fields']['language_1']['field'] = 'language';
+  $handler->display->display_options['fields']['language_1']['label'] = 'Source Language';
   /* Field: Entity translation: Language */
```
Comment (blisteringherb) [inline lib/modules/dosomething/dosomething_user/dosomething_user.views_default.inc:313]: Was switching out these fields supposed to be a part of this PR?

**Comment by itsjoekent** [comment]
Nope, good catch, not sure how this got in here.

**Comment by itsjoekent** [comment]
@blisteringherb fixed!

**Comment by blisteringherb** [comment]
:+1:

**Pull request merged**